mrmustard/lab_dev/circuit_components.py

```diff
@@ -104,6 +107,45 @@ def __init__(
                 if self._representation:
                     self._representation = self._representation.reorder(tuple(perm))
 
+    def serialize(self) -> tuple[dict[str, Any], dict[str, ArrayLike]]:
```

Comment:
I think we should rename these as well but not sure if `to_dict` still works here?

Reply:
I think the use-case is too bespoke here for a general name - the function creates a tuple of dicts, one serializable and one not. I believe its only use is for serialization, so I couldn't think of a better name personally

Reply:
`to_serialize`? lol. Maybe just make it a hidden method since we dont expect users to call it 👍

Reply:
done!